### PR TITLE
Slight cleanup, horrible debug delayer

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

**Comment by ryanpbrewster** [comment]
:scream:

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala [hunk 1]
```diff
@@ -172,7 +172,8 @@ class KifiSiteRouter @Inject() (
   private def serveWebAppToUser2(dependencies: Seq[PreloadRequest] = Seq.empty)(implicit request: MaybeUserRequest[_]): Result = request match {
     case ur: UserRequest[_] =>
       userIpAddressCommander.logUserByRequest(ur)
-      app(dependencies).discardingCookies(PostRegIntent.discardingCookies: _*)
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:175]: @camhashemi This was actively clearing out these cookies even when they were not set. Now we only clear set ones out.

Comment (camhashemi) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/routing/KifiSiteRouter.scala:175]: ah, I noticed there was some weird behavior in the headers, but assumed you couldn't clear cookies that weren't set. thanks!

**Comment by andrewconner** [comment]
Shelving this for now.

**Pull request closed without merging**